File: solutions/typescript/src/main2.ts
import * as fs from 'fs'

const main = (args: string) => {
  const [a, b] = args.split(' ').map((i) => Number(i))
  const answer = (a * b) % 2 === 0 ? 'Even' : 'Odd'
  console.log(answer)
}

main(fs.readFileSync('/dev/stdin', 'utf-8'))

File: solutions/rust/main2.rs
use proconio::input;

fn main() {
    input! {
        a: u32,
        b: u32,
    }

    let answer = if a * b % 2 == 1 {
        "Odd"
    } else {
        "Even"
    };
    println!("{}", answer);
}
